test/test_markov.py

```diff
@@ -156,3 +156,9 @@ def test_notify_admin(mock_settings, mock_bot):
         mock_settings.ADMIN_CHAT_ID,
         message
     )
+
+
+@mock.patch('markov.bot')
+def test_help(mock_bot, message):
+    markov.help(message)
+    assert mock_bot.reply_to.mock_model.called_once_with(message)
```

Comment:
i think just `assert mock_bot.reply_to.called is True` is enough

Reply:
Ah yup that makes sense, I agree.